lib/mockingjay.dart

```diff
@@ -0,0 +1,6 @@
+/// An experimental package that makes it easy to mock, test and verify
+/// navigation calls in Flutter.
+library mockingjay;
+
+export 'src/matchers.dart';
+export 'src/mock_navigator.dart';
```

Comment:
```suggestion
export 'src/mockingjay.dart';
```
minor but should we rename?

Reply:
I was wondering the same thing. I opted for `mock_navigator.dart` since nothing in that class is actually called `class Mockingjay` or something along those lines. Could really go either way 👍🏻

Reply:
yeah that makes sense I'm fine leaving as is 👍